example-library/src/main/java/uk/gov/gchq/palisade/example/config/ExampleUserConfiguration.java

```diff
@@ -46,20 +43,20 @@ public ExampleUserConfiguration() {
      * Constructor with 1 argument for an example implementation
      * of the {@link UserConfiguration} interface
      *
-     * @param users     a list of objects implementing the {@link uk.gov.gchq.palisade.service.UserCacheWarmerFactory} interface
+     * @param users     a list of objects implementing the {@link UserPrepopulationFactory} interface
```

Comment:
Can this be turned back to using the fully qualified name so the import can be removed